### ISSUE TITLE
Image in docker

### ISSUE THREAD

**Issue by talesam** [ogioncz/flarum-webhooks-telegram-bridge#1]
Hi, isn't there a docker image of this app?

**Comment by jtojnar** [comment]
Hi. I have not created one as I have no need for it. Feel free to create one.

**Comment by talesam** [comment]
https://hub.docker.com/repository/docker/talesam/big-bot-flarum-telegram

**Comment by jtojnar** [comment]
Thanks. I suppose the link is https://hub.docker.com/r/talesam/big-bot-flarum-telegram since the one you sent wants me to log in to docker hub.

**Comment by talesam** [comment]
That, I had sent you the link logged in, I'll create another image with the name without "big" and send it to the hub, for ARM64 and X86, then I'll post it here, if you want to publish it, feel free, the image is 100% functional, and with the config.php file inside, just assemble and configure it, I'll create a docker-compose to make it easier too.

**Comment by talesam** [comment]
I have a doubt! I believe container for 6 different architectures, but I have a question, do you need a web server to run your app or not? The image has an apache server running with php and composer, if you don't need the web server I can remove apache to make it smaller.

**Comment by jtojnar** [comment]
Yeah, this project works as [webhooks](https://en.wikipedia.org/wiki/Webhook) end point so it needs a HTTP server. In a pinch, you could do with the [web server built into PHP](https://www.php.net/manual/en/features.commandline.webserver.php) but PHP docs advise against it.
